doc/source/whatsnew/v0.21.0.txt

```diff
@@ -201,3 +201,4 @@ Categorical
 Other
 ^^^^^
 - Bug in :func:`eval` where the ``inplace`` parameter was being incorrectly handled (:issue:`16732`)
+- The ``Series`` constructor with no arguments would have an index like ``Index([], dtype='object')`` instead of ``RangeIndex(start=0, stop=0, step=1)``
```

Comment:
make a less verbose comment, like:

``Series`` constructor will default to construct a RangeIndex, rather than an ``Index`` when constructed as empty.